site/en/docs/workbox/caching-strategies-overview/index.md

```diff
@@ -111,7 +111,7 @@ which is the URL for the network request currently being handled by the `fetch`
 which is the request method (e.g., `GET` or `POST`).
 - [`mode`](https://developer.mozilla.org/en-US/docs/Web/API/Request/mode),
```

Comment:
```suggestion
- [`mode`](https://developer.mozilla.org/docs/Web/API/Request/mode),
```

Reply:
Managed to fix this (and other offenders in an earlier doc) in a commit, so this is outdated currently.